library/std/src/os/fd/owned.rs

```diff
@@ -342,3 +342,30 @@ impl From<OwnedFd> for crate::net::UdpSocket {
         ))))
     }
 }
+
+#[unstable(feature = "io_safety", issue = "87074")]
+/// This blanket impl allows implementing custom traits that require `AsFd` on Arc.
```

Comment:
"Custom traits" seems redundant to me. What makes a particular trait custom? I think you just mean "traits"?

Reply:
I meant something like "traits not defined in the standard library". But I agree it's unclear and not super useful, I'll just say "traits".